HousingRepairsSchedulingApi/Startup.cs

```diff
@@ -48,19 +49,9 @@ public void ConfigureServices(IServiceCollection services)
                 return new SOAPClient(binding, new EndpointAddress(apiAddress));
             });
 
-            services.AddTransient<IDrsService, DrsService>();
-
-            services.AddTransient<IAppointmentsGateway, DrsAppointmentGateway>(
-                sp =>
-                {
-                    var drsOptions = sp.GetRequiredService<IOptions<DrsOptions>>();
-                    var appointmentSearchTimeSpanInDays = drsOptions.Value.SearchTimeSpanInDays;
-                    var appointmentLeadTimeInDays = drsOptions.Value.AppointmentLeadTimeInDays;
-                    var maximumNumberOfRequests = drsOptions.Value.MaximumNumberOfRequests;
-                    return new DrsAppointmentGateway(sp.GetService<IDrsService>(),
-                        5, appointmentSearchTimeSpanInDays, appointmentLeadTimeInDays, maximumNumberOfRequests);
-                }
-            );
+            services.AddTransient<IDrsService, DrsService>();

```

Comment:
This isn't logging - why has this changed?

Reply:
In line 82 when DrsAppointmentGateway is instantiated the instance of logger (line 72) is passed in. Do you think it'd be better not to have a separate InitializeAppointmentsGateway function?

Reply:
I think I see what you mean, I'll try and refactor

Reply:
@Asura5 I've made a change now and pushed, checks are green, please see the below:
https://github.com/LBHackney-IT/HousingRepairsSchedulingApi/pull/17/commits/69017aff0086d5c3483cc7a074bb963e826a5e03